kernel/packages/shared/apis/PortableExperiences.ts

```diff
@@ -0,0 +1,64 @@
+import { registerAPI, exposeMethod } from 'decentraland-rpc/lib/host'
+import {
+  spawnPortableExperienceScene,
+  getPortableExperience,
+  PortableExperienceHandle,
+  killPortableExperienceScene
+} from 'unity-interface/portableExperiencesUtils'
+import { ExposableAPI } from './ExposableAPI'
+import { ParcelIdentity } from './ParcelIdentity'
+
+type SpawnPortableExperienceParameters = {
+  urn: string
+}
+
+@registerAPI('PortableExperiences')
+export class PortableExperiences extends ExposableAPI {
+  /**
```

Comment:
We should copy these comments into the declaration, so that SDK users understand what each of these methods do